R/biting_process.R

```diff
@@ -118,6 +119,7 @@ simulate_bites <- function(
 
     lagged_eir[[s_i]]$save(n_infectious * a, timestep)
     species_eir <- lagged_eir[[s_i]]$get(timestep - parameters$de)
+    renderer$render(paste0('EIR_', species_name), species_eir, timestep)
     EIR <- EIR + species_eir
```

Comment:
Great idea to disaggregate. Will it be easy for people to calculate overall EIR? Is this EIR the overall, or will people add(?) them?

Reply:
Yup, total EIR is just the sum of each species EIR